modules/apps/address/address-web/src/main/resources/META-INF/resources/js/CountryRegionDynamicSelect.js

```diff
@@ -73,6 +76,28 @@ function CountryRegionDynamicSelect({
 	]);
 }
 
+function updateRegionRequired(regions, namespace) {
+	const regionRequiredWrapper = document.getElementById(
+		`${namespace}regionRequiredWrapper`
+	);
+	const formValidator = Liferay.Form.get(`${namespace}fm`).formValidator;
```

Comment:
can we add an 
```if (!regionRequiredWrapper) { return }```
in this way if this script is called in some place where there is no the regionRequiredWrapper doesn't fail